examples/docs_snippets/docs_snippets/concepts/assets/auto_materialize_sensor.py

```diff
@@ -0,0 +1,9 @@
+from dagster import AssetSelection, AutoMaterializeSensorDefinition, Definitions
```

Comment:
Ideally we would have a test that at least makes sure this `Definitions` object can be created successfully.